.github/workflows/ci.yml

```diff
@@ -47,9 +47,12 @@ jobs:
         python setup.py install
       env:
         AIOHTTP_NO_EXTENSIONS: 1
-    - name: Run linters
+    - name: Mypy
       run: |
         make mypy
+    - name: Flake8
+      run: |
+        flake8
```

Comment:
This should be integrated into pre-commit instead.

Reply:
Hmm, it already is. So, why does flake8 pass without errors on my first commit?

Reply:
Maybe it runs in a slightly different env (because it has its own venv set up by pre-commit).

Reply:
It's because you have to add this plugin as an additional dependency to the pre-commit hook config. It doesn't use these pip-tools' lock files. Well, unless you remake how the hook is set up it won't.

Reply:
This is why it's hard to integrate the pin files into pre-commit w/o an external wrapper: https://github.com/pre-commit/pre-commit/issues/1963.